src/main.jsx

```diff
@@ -116,21 +116,20 @@ function App() {
     const [flip, setFlip] = useState(false)
     const pane = useRef()
 
-    function switchToTab(index, pushState) {
+    function switchToTab(index, pushState, prevTabIndex) {
         if (index === paneIndex || index === tabIndex) return
-
-        console.info("switching to tab", index)
-
         setRotation(rotation - 180)
-        setTabIndex(index)
+
+        console.info("switching to tab", index, "from ", prevTabIndex)
 
         if (pushState)
             history.pushState(null, tabs[index].name, tabs[index].slug)
 
         setTimeout(() => {
+            setTabIndex(index)
             setPaneIndex(index)
             setFlip(!flip)
-        }, 300) // half the animation time
+        }, prevTabIndex === 0 ? 175 : 300) // half the animation time
```

Comment:
leaving info page doesn't need as much of a timeout and benefits from having its contents hidden quicker. Also part of why transitioning from the tab sometimes shows the text flip.